config/autoload/local.php.dist

```diff
@@ -1,3 +1,9 @@
 <?php
+
+use Zend\ConfigAggregator\ConfigAggregator;
+
 return [
+    // Cache the configuration: Recommended for production
+    // Closures used somewhere in the configuration make caching impossible
+    ConfigAggregator::ENABLE_CACHE => false,
```

Comment:
You want this in local.php since you only want this enabled on your production server and an user should set it in that file. Also you don't want caching enabled by default since that really confuses people during development. I've seen a lot of issues with this for ZF3 in #zftalk.

It's also in development.config.php to make sure it is set false in development mode (in case it's enabled in local.php). Since development.config.php is loaded last, it will overwrite all previous states for this setting.

It shouldn't confuse new users since development.config.php.dist should not be edited and is meant for the zf-dev-mode tool only. Maybe a warning on top of that file might make this clear.